README.md

```diff
@@ -210,6 +210,7 @@ Feel free to check the [issues page](https://github.com/EmekaEmmanuel/
 math_magiciansReact/issues).
 
```

Comment:
- [ ] [optional] It would be nice if you checked your issue page link code and ensure its displayed properly.